box-share-sample/src/main/java/com/box/androidsdk/sample/MainActivity.java

```diff
@@ -176,7 +178,7 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
         if (requestCode == REQUEST_CODE_SHARE_LINK){
             if (data != null){
                 // update current data to the latest one returned from the shared link creation.
-                mSampleFolder = (BoxFolder) BoxSharedLinkActivity.createResultInterpreter(data).getBoxItem();
+                mSampleFolder = (BoxFolder) new CollaborationUtils.ResultInterpreter(data).getBoxItem();
```

Comment:
I think it is cleaner to include the static helper here as developers are interacting with a specific activity. I'm not sure it is clear that they should use a class in CollaborationUtils.